README.md

```diff
@@ -0,0 +1,58 @@
+# navitia_model [![Build Status](https://travis-ci.org/CanalTP/navitia_model.svg?branch=master)](https://travis-ci.org/CanalTP/navitia_model)
+
+- [Navitia Transit Feed Specification (NTFS)](https://github.com/CanalTP/navitia/blob/dev/documentation/ntfs/ntfs_fr.md) modeling
+- Conversion [GTFS](http://gtfs.org/) to [NTFS](https://github.com/CanalTP/navitia/blob/dev/documentation/ntfs/ntfs_fr.md)
+
```

Comment:
Small description here, something like:

`navitia_model` is a rust crate to manage transit data. Its model corresponds to the one used in [navitia](https://github.com/CanalTP/navitia). This repository also provides (incomplete) GTFS to NTFS and (soon) NTFS to GTFS conversion.